data/mods/blazemod/blaze_blob_parts.json

```diff
@@ -684,22 +693,31 @@
   {
     "id": "gray_tank",
     "type": "vehicle_part",
-    "name": "cocoon (25L)",
+    "name": "cocoon (30L)",
     "item": "gray_tank",
-    "location": "fuel_source",
+    "//location": "fuel_source",
```

Comment:
This is not how CDDA does comments in JSON and this is not how CDDA removes a field from JSON. Also, standard vehicles can't put a 10L, 20L, and 60L tank on the same tile, so don't change how that works for blob parts.